drive_cli/actions.py

```diff
@@ -447,7 +447,7 @@ def history(date, clear):
                 for date in History:
                     history = History[date]
                     for i in history:
-                        click.secho(date + "  " + i, fg='yellow', bold=True)
+                        click.secho(date + "  " + i, fg='yellow', bold=True
```

Comment:
@raghav-dalmia Did you try to run this locally? you've missed to close with `)`.

Reply:
Yes, I run this on local. This might happens when I was resolving merge conflicts. Sorry about that.